pubspec.yaml

```diff
@@ -3,12 +3,12 @@
 # license that can be found in the LICENSE file.
 
 name: basics
-version: 0.8.0
+version: 0.9.0
```

Comment:
is `0.9.0` right here? Feels like this is an implementation-only change, it doesn't change the public API or the performance of the internals.

Maybe `0.8.1` is more appropriate?

Reply:
I chose 0.9.0 because I bumped the Dart SDK requirement to 2.17 for `strict-casts`, and I thought that the SDK requirement could be considered breaking.  But looking at the semver spec, I guess that's not true.  I'll change it to 0.8.1

Reply:
Ah, makes sense, SGTM